.travis.yml

```diff
@@ -47,15 +47,9 @@ matrix:
 
 os:
   - linux
-  - osx
 
 before_install:
-  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then PATH=/usr/local/opt/ccache/libexec:$PATH; fi
-  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
-      export HOST_OS="darwin";
-    else
-      export HOST_OS="$TRAVIS_OS_NAME";
-    fi
+  - export HOST_OS="$TRAVIS_OS_NAME";
```

Comment:
And to be strictly pedantic, I guess it doesn't hurt to leave all this `before_install` stuff in here, "just in case"? The only bit we _actually_ need to remove is `os: osx` from the build matrix?

Reply:
We can always revert the commit. The information will remain in git